third_party/2and3/boto/connection.pyi

```diff
@@ -70,7 +70,9 @@ class AWSAuthConnection:
     @property
     def aws_access_key_id(self): ...
     @property
-    def gs_access_key_id(self): ...  # type: Any
+    def gs_access_key_id(self):
+      # type: (...) -> Any
+      ...
```

Comment:
Maybe while you are at it, you can change this to the Python 3 style annotation?